### PR TITLE
compressed.d: stress that the headers are not modified

### PR DESCRIPTION
**Pull request by bagder**
Suggested-by: Michael Osipov
Bug: https://github.com/curl/curl/issues/5182#issuecomment-611638008

### CODE REVIEW

**Comment by jay** [comment]
I think it is implied. Seems like a long winded way to say it: _Associated received response headers may of course not match the processed, uncompressed body,_ How about just _Headers are not modified._

**Comment by bagder** [comment]
I like that. Shorter and more to the point!

File: docs/cmdline-opts/compressed.d [hunk 1]
```diff
@@ -3,5 +3,7 @@ Help: Request compressed response
 Protocols: HTTP
 ---
 Request a compressed response using one of the algorithms curl supports, and
-save the uncompressed document.  If this option is used and the server sends
-an unsupported encoding, curl will report an error.
+save the uncompressed document. Headers are not modified.
```
Comment (jay) [inline docs/cmdline-opts/compressed.d:6]: Another suggestion , _save the uncompressed document_ is not always right, the content may not be saved.
```suggestion
automatically decompress the response. Headers are not modified.
```

Comment (bagder) [reply docs/cmdline-opts/compressed.d:6]: Yeps, that's better!

**Pull request closed without merging**